Added an indicator of "below the fold" unread entries

fmunch opened the pull request:
Unread entries might appear far down in the list of entries for many reasons. For example, when marking a source as read, the recent entries list will be a mix of read / unread entries.
When marking everything as read we might have missed a few unread entries because they were too far down in the list.

This PR adds an indicator displaying the number of unread entries below the fold:

![Indicator of below the fold unread entries](http://i.imgur.com/WWiBYnp.png)

It only appears on desktop browsers (width > 1024px), and requires the `floating_unread_count_enabled=true` config parameter to be shown (disabled by default).

It might display `?` when the number of unread entries is unknown, for example when using the search feature.

jtojnar commented:
Looks great. Just few comments:

* Could you rebase it to get rid of the “temporary code” commit?
* Could you add the config option into defaults.ini? (It is to make the default value more obvious.)
* Other boolean options use `0`/`1`, could you use this convention?
* Could you also update documentation (_docs/website/index.html)?
* If the tag/source has no unread items, the widget displays the number of all unread items from all the categories. The same happens on the Settings page.
* Why display the question mark and not hide the widget altogether when the number of unread is unknown?
* Could you add `aria-hidden="true"` attribute to the widget? The number might confuse screen reader users.

niol commented:
Interesting. Why a config option? Who wouldn't want this?

fmunch commented:
I fixed the issue when the tag/source has no unread items, and added the `aria-hidden` attribute, I will try to push that later today.
The config option is not really useful, I added it just to be sure that it could be disabled in case of a bug or something. I will remove it.
I added the question mark to clearly show that there _might_ be unread entries, which is different from when we _know_ that everything is read after that. But I can remove it if it is not self-explanatory.

fmunch commented:
Ok, so
* The config option is removed
* I added the `aria-hidden` attribute
* The bug when the tag / source had no unread entry should be fixed
* The unread count on the settings view should now be hidden
* Everything was rebased in a single commit

It can still show a question mark on a few views, should I remove this behavior too?

SSilence commented:
Thanks for this pr. A really nice feature. Can you fix the conflicts? Then I will merge this.

Thanks a lot!!

fmunch commented:
Should be fine now.
